nixos/modules/services/networking/nordvpn.nix

```diff
@@ -0,0 +1,99 @@
+{
+  config,
+  lib,
+  pkgs,
+  proc-ps,
+  ...
+}:
+
+let
+  cfg = config.services.nordvpn;
+  nordvpnPkg = cfg.package;
+in
+
+{
+  options.services.nordvpn = {
+    enable = lib.mkEnableOption "Enable NordVPN";
+    package = lib.mkPackageOption pkgs "nordvpn" { };
+  };
+
+  config = lib.mkIf cfg.enable {
+
+    users.groups.nordvpn = { };
+
+    users.users.nordvpn = {
```

Comment:
I see that the user and group `nordvpn` are hardcoded. Don't you think we should let the end-user decide which user and group to set? Perhaps you should create two new config options `user` and `group` with their default value set to `nordvpn`.